source/modules/MEI.shared.xml

```diff
@@ -6635,7 +6628,6 @@
           <rng:choice>
             <rng:ref name="imprint"/>
             <rng:ref name="extent"/>
-            <rng:ref name="biblScope"/>
```

Comment:
@pe-ro Is the removal of biblScope here correct? Is it not needed in monogr anymore?

Reply:
@musicEnfanthen, good eye!  However, I think the removal of `<biblScope>` is appropriate.